lib/sign.js

```diff
@@ -57,11 +57,11 @@ function sign(options, config) {
       }
     });
     if (missingOptions.length) {
-      console.error();
+      console.error(); // eslint-disable-line no-console
```

Comment:
I think all these do need to be direct console calls. I forget why exactly but there was some reason.